src/config/socialAuth.js

```diff
@@ -0,0 +1,41 @@
+import passport from 'passport';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+import { OAuth2Strategy as googleStrategy } from 'passport-google-oauth';
+import { Strategy as twitterStrategy } from 'passport-twitter';
+import { socialAuth } from '../routes/api/users/auth';
+
+passport.serializeUser((userDetail, done) => done(null, userDetail));
+passport.use(new FacebookStrategy({
+  clientID: process.env.facebookClientID,
+  clientSecret: process.env.facebookClientSecret,
+  callbackURL: 'https://ivy-ah-backend.herokuapp.com/api/v1/users/facebookRedirect',
```

Comment:
Avoid hardcoding values like this in your code.
Move this to a config file.

Reply:
Ok